test/con_cache_test.exs

```diff
@@ -184,6 +184,24 @@ defmodule ConCacheTest do
     end)
   end
 
+  test "get_or_store on bag" do
+    with_cache([ets_options: [:bag]],fn(cache) ->
+      assert ConCache.get_or_store(cache, :a, fn() -> 1 end) == 1
```

Comment:
I didn't get whats the intention here. It would be to support `get_or_storer` to insert multiple values in a single call when the ets table is a bag type?

I don't know if this would be a good idea. First, since I can store any term, `get_or_store` should work the same way for set and bag types. If we make that change, when calling `ConCache.get_or_store(cache, :a, fn() -> [1, 2, 3] end)` for a set type it will insert `[1, 2, 3]` as a term, calling the same way with a bag type will insert multiple values.

My suggestion would be to have a new `get_or_store` where the third parameter is a list not a function, that way, when the user passes a function we insert the value as it is, when it is a list whe insert it as multiple values(if the table is a bag). What do you think?